### PR TITLE
Updated link for OS specific installation

### PR DESCRIPTION
**Pull request by marcello-goccia**
Added the link to the operating system-specific installation documentation in the sections 'System Dependencies'.

### CODE REVIEW

**Comment by traversaro** [comment]
Friendly ping @marcello-goccia

File: doc/cmake-options.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ This profile is enabled by the `ROBOTOLOGY_ENABLE_CORE` CMake option.
 
 ### System Dependencies
 The steps necessary to install the system dependencies of the Core profile are provided in
-operating system-specific installation documentation.
+[operating system-specific installation documentation](https://github.com/robotology/robotology-superbuild#linux-from-source-with-dependencies-provided-by-apt).
```
[the remark this answers is not in the document]
Comment (marcello-goccia) [reply doc/cmake-options.md:115]: Looks good, thank you!

**Review by marcello-goccia — COMMENTED** [review]
Looks good, thank you!

File: doc/cmake-options.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (traversaro) [reply doc/cmake-options.md:115]: Hi @marcello-goccia, if you think that the change is ok feel free to apply it to the PR, thanks!

**Comment by traversaro** [comment]
Thanks @marcello-goccia !

**Pull request merged**